.gitignore

```diff
@@ -0,0 +1,3 @@
+pillar/config.sls
```

Comment:
Last question: why is this here? This PR does not seem to affect this file at all. I guess this line should just be removed.